src/inventory.gql

```diff
@@ -16,12 +16,44 @@ type ProductItem {
 """A simple UUID parser"""
 scalar UUID
 
+type PaginatedProductItem {
+  nodes: [ProductItem!]
+  totalCount: Int!
+  hasNextPage: Boolean!
+}
+
 type Query {
-  productItems: [ProductItem!]!
+  productItems(findProductItemsInput: FindProductItemArgs!): PaginatedProductItem!
   countProductItems(productVersionId: UUID!): Int!
   productItem(_id: UUID!): ProductItem!
 }
 
+input FindProductItemArgs {
+  skip: Int! = 0
```

Comment:
this could cause issues, as our schema defines these as optional, and declares the default values in code
however, in particular if first is not given, it should return all
orderBy should default to ID and ASC